packages/trip-form/src/types.ts

```diff
@@ -0,0 +1,113 @@
+import { FunctionComponent, ReactNode } from "react";
+import { StyledComponent } from "styled-components";
+
+/**
+ * Describes the parameters for the onQueryParamChange event.
+ */
+export interface QueryParamChangeEvent {
+  [key: string]: string | number | boolean;
+}
+
+/**
+ * Describes the text/value pair for each query parameter option.
+ */
+export interface QueryParamOptions {
+  text: string;
+  value: SVGForeignObjectElement;
```

Comment:
Is a QueryParam value always an SVG?

Reply:
Good catch... must have been an auto-complete thing.